enterprise/frontend/src/embedding-sdk/components/public/MetabotQuestion/MetabotChatEmbedding.tsx

```diff
@@ -102,27 +105,15 @@ export const MetabotChatEmbedding = ({
     dispatch(resetConversationId());
   }, [dispatch]);
 
+  useEffect(() => {
+    const normalizedMessages = metabot.userMessages.filter(
+      (message) => message !== METABOT_RESULTS_MESSAGE,
+    );
+    onMessages(normalizedMessages);
+  }, [metabot.userMessages, onMessages]);
+
   return (
     <Box className={Styles.container} data-testid="metabot-chat">
-      {metabot.userMessages.length > 0 && (
```

Comment:
Text bubble is removed to match the design. It used to look like this.
![image](https://github.com/user-attachments/assets/9132a9f1-59e8-4a7e-8ef0-e5b66ea9f3e8)